styles/globals.css

```diff
@@ -26,7 +26,7 @@
 
 /* Apply additional styles to .mapboxgl-ctrl-geocoder */
 .mapboxgl-ctrl-geocoder {
-  @apply h-14 min-w-full bg-ocf-black !important;
+  @apply h-14 min-w-full rounded-lg bg-ocf-black !important;
```

Comment:
I'm pretty sure this class represents the wrapper that contains the geocoder—my concern rn is that making this rounded might cause problems in the future if, for some reason, the geocoder itself needs to be changed. The background color is probably what's causing the black underlay—maybe changing that instead of changing the container to fit its contents would be better

```suggestion
  @apply h-14 min-w-full bg-transparent !important;
```

Reply:
yep this makes sense!